components/service/glean/src/test/java/mozilla/components/service/glean/private/StringSetMetricTypeTest.kt

```diff
@@ -38,33 +38,33 @@ class StringListMetricTypeTest {
         )
 
         // Record two lists using add and set
-        stringListMetric.add("value1")
-        stringListMetric.add("value2")
-        stringListMetric.add("value3")
+        stringSetMetric.add("value1")
+        stringSetMetric.add("value2")
+        stringSetMetric.add("value3")
 
         // Check that data was properly recorded.
-        val snapshot = stringListMetric.testGetValue()
+        val snapshot = stringSetMetric.testGetValue()
         assertEquals(3, snapshot.size)
-        assertTrue(stringListMetric.testHasValue())
-        assertEquals("value1", snapshot[0])
-        assertEquals("value2", snapshot[1])
-        assertEquals("value3", snapshot[2])
+        assertTrue(stringSetMetric.testHasValue())
+        assertEquals("value1", snapshot.elementAt(0))
```

Comment:
using `elementAt` on a `Set` concerns me a bit, I'm not sure that Set is guaranteed to be in any order.  From the [docs](https://developer.android.com/reference/java/util/Set#iterator()) it looks like the implemented iterator doesn't return in any particular order.  Frankly I'm surprised that function is even exposed on the `Set`.  Since this is a set, it would be best to use [`contains`](https://developer.android.com/reference/java/util/Set#contains(java.lang.Object)) or [`containsAll`](https://developer.android.com/reference/java/util/Set#containsAll(java.util.Collection%3C?%3E)) instead.  This applies to everywhere `elementAt` is being used.